src/main/java/org/patchmanager/Main.java

```diff
@@ -18,125 +18,38 @@ public class Main {
-
-
-      try {
-        LOGGER.info("Checking if the patch is in correct form");
-        patchInputChecker(commandLine);
-      } catch (Exception e) {
-        LOGGER.fatal("Patch does not fit the criteria");
-        printHelpCmd(optionsList);
-        System.exit(-1);
-      }
-      patchInput = commandLine.getOptionValue(PATCH);
-    } catch (Exception e) {
-      LOGGER.fatal(e.getMessage());
-    }
-    LOGGER.info("Trying to authorize");
-    DotEnvUser dotEnvUserObj = new DotEnvUser();
-
-
-    if (!checkAuth(dotEnvUserObj.email, dotEnvUserObj.api)) {
-      LOGGER.fatal("Couldn't authorize, check credentials");
-      System.exit(-1);
-    }
-
-    LOGGER.info("Defining version values");
-    //There are 2 versions that are used, a higher one e.g. 9.8.1 and a lower one e.g. 4.8.1
-    //Split by numbers
-    String[] splitVersionByNumbers = versionInput.split("(\\D+)");
-    String versionHigher = String.join(".", splitVersionByNumbers[0], splitVersionByNumbers[1], splitVersionByNumbers[2]);
-    String lowerFirstNoByFive = String.valueOf(Integer.parseInt(splitVersionByNumbers[0]) - 5);
-    String versionLower = String.join(".", lowerFirstNoByFive, splitVersionByNumbers[1], splitVersionByNumbers[2]);
-
-    LOGGER.info("Creating a variable for today's date");
-    Date date = new Date();
-    SimpleDateFormat simpleDateFormatObj = new SimpleDateFormat("yyyyMMdd");
-    String strDate = simpleDateFormatObj.format(date);
-
-    LOGGER.info("Deciding on filename");
-    String fileName = fileNameDecider(patchInput, versionInput);
-
-    LOGGER.info("Trying to send a Http Request to the API and get a response");
-    HttpRequestAndResponse httpRequestAndResponse = new HttpRequestAndResponse(labelInput);
-    String parsedJiraIssues = parseJiraIssues(httpRequestAndResponse.response.body(), versionLower, patchInput);
-    if (!parsedJiraIssues.isEmpty()) {
-      StringBuilder finalStr = new StringBuilder();
-      finalStr.append(writeIntro(patchInput, versionLower, versionHigher, versionInput, strDate))
-          .append(parsedJiraIssues)
-          .append(writeOutro(patchInput, versionHigher));
-      writeToFile(finalStr.toString(), fileName);
-    } else {
-      LOGGER.fatal("No issues found with that label");
-      System.exit(-1);
-    }
+    mainMenu();
     LOGGER.info("Finishing the main function");
-     **/
+    long finish = System.currentTimeMillis();
+    double timeElapsed = finish - start;
+    System.out.println(timeElapsed/1000);
+    System.out.println("seconds passed");
   }
-  //+gitignore
+  //make password masked
   //make parse string response shorter
   //check style
-  //+parse check first 15 chars only
   //check performance of long regex
-  //+try catch null option missing
-  //+mockito
-  //+api test
-  //+change name of classes to capital starting letter
-  //+display path and success
-  //+two chars
-  //+remove [] things and such
-  //+log4j
+  //username sifre input
```

Comment:
Please leave the comments at the related code/function/class with `// TODO:` annotation and remove unnecessary comments.

Reply:
Done

Reply:
Could you please self-review your PR and update the entire diff according to PR comments?